Assets/Scripts/CameraScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraScript : MonoBehaviour
+{
+	
+	Vector3 defaultCameraPosition;
+	Quaternion defaultCameraRotation;
+	Transform cameraTransform;
+	
+    // Start is called before the first frame update
+    void Start()
+    {
+		cameraTransform = GetComponent<Transform>();
+        defaultCameraPosition = cameraTransform.position;
+		defaultCameraRotation = cameraTransform.rotation;
+    }
+	
+	public void Reset()
+	{
+		cameraTransform.position = defaultCameraPosition;
+		
+		cameraTransform.rotation = Quaternion.Slerp(cameraTransform.rotation, defaultCameraRotation, 0.5F);

```

Comment:
Will this work? Isn't cameraTransform just a copy, not a reference to the camera's transform? I'm not seeing it ever reset

Reply:
Just saw this-- cameraTransform is a reference to the camera's transform. I think the problem before was that I was accessing this Reset() function by using GetComponent<>() in CatBehaviorNodes.cs. However, that is a MonoBehaviour function, and none of the nodes inherit from MonoBehaviour, so for some reason it was silently failing.

I called the Reset() function inside Cat.cs and it worked. It should work as intended now in my latest commit!